settings.php

```diff
@@ -1186,7 +1186,7 @@ function get_FTL_data($arg)
                         <div class="col-md-12">
                             <div class="box box-warning">
                                 <div class="box-header with-border">
-                                    <h3 class="box-title">Danger Zone!</h3><br/>
+                                    <h3 class="box-title">Danger Zone - Use with Caution!</h3><br/>
```

Comment:
`Danger Zone!` already conveys the message of caution, so no need to add more to it.